src/commonMain/kotlin/arrow.exact/Exact.kt

```diff
@@ -2,27 +2,21 @@ package arrow.exact
 
 import arrow.core.Either
 
+public interface Exact<A, out R> : ExactEither<ExactError, A, R>
```

Comment:
@nomisRev Q: Why do we have to right thess explicit `public`?

Reply:
We setup _explicit API_ in Arrow, https://github.com/arrow-kt/arrow-gradle-config/blob/e9298c1f7a0efb1e72194381aedd1a366094cad1/arrow-gradle-config-kotlin/src/main/kotlin/io.arrow-kt.arrow-gradle-config-kotlin.gradle.kts#L21

This is a bit saner for library development.

Reply:
Makes sense. Thank you!